pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <modelVersion>4.0.0</modelVersion>
   <artifactId>rspace-core-model</artifactId>
-  <version>2.2.0</version>
+  <version>RSDEV-310-SNAPSHOT</version>
```

Comment:
We usually add the ticket ID to the artefact i.e. `rspace-core-model-RSDEV-310` as maven can be quite fussy about versions. Probably won't cause you any issues since it's a one-off for your feature branch but just mentioning it in case.